test/environment_unittest.cc

```diff
@@ -42,6 +49,59 @@ class TemporaryFile {
  private:
   boost::filesystem::path path_;
 };
+
+// Starts a server in a separate thread, allowing it to choose an
+// available port.
+class FakeServerThread {
+ public:
+  FakeServerThread()
+    : server_(Server::options(handler_).address("127.0.0.1").port("")) {
+    server_.listen();
+    thread_ = std::thread([this] { server_.run(); });
+  }
+
+  void SetResponse(const std::string& path, const std::string& response) {
+    handler_.path_responses[path] = response;
+  }
+
+  std::string HostPort() {
```

Comment:
Just curious, why `HostPort` and not `GetUrl`?

Reply:
Changed to GetUrl.